### DISCUSSION TITLE
Scrolling the mouse changes the volume in the flyout, even with the appropriate settings.

### DISCUSSION THREAD

**Discussion by Leo-skr** [File-New-Project/EarTrumpet#1625] [Ideas]
### Summary

I turned off both options for scroll wheel volume control (One for when hovering the icon and the other one for the flyout) and on the flyout it still changes the volume when scrolling.

### Steps to reproduce

- Set setting "Use the scroll wheel to change volume while the flyout is open" under Mouse settings to off
- Open flyout and scroll while above a volume slider

### EarTrumpet version

2.3.0.0

### Windows version

Microsoft Windows [Version 10.0.22631.4037]

**Comment by riverar** [comment]
Those settings apply to scrolling outside the flyout. I think it's a reasonable feature request to add an option to stop taking scroll input in the flyout too. Will forward it to discussions.

**Reply by octocatsup** [reply]
Has there been any movement or updates on this feature request?
